README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
`meta/main.yml` clearly states supported (and tested) versions. in general, i have been trying to make roles version-agnostic. why are you trying to be so specific? according to your suggestion, we must maintain every version of Debian, such as 8.6, 8.7, and so on just for nothing. i do not see the point.